README.md

```diff
@@ -36,7 +36,7 @@
     <a href="https://github.com/DucktapeEngine/Ducktape/issues/new?assignees=&labels=&template=feature_request.md&title=">Request Feature</a>
 </p>
 
-> **Note: The original Ducktape Engine is going under a rewrite with a completely different goal in mind.**
+> **Note: The original Ducktape Engine is undergoing a rewrite with a completely different goal in mind.**
```

Comment:
I think it's appropriate to remove this line now, since it's been about 6 months sine the rewrite started.